nav2_bringup/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ find_package(navigation2 REQUIRED)
 
 nav2_package()
 
+install(DIRECTORY config DESTINATION share/${PROJECT_NAME})
```

Comment:
This can be removed since it doesn't exist anymore :smile: I think that's why CI failed

Reply:
ups, https://github.com/ros-planning/navigation2/pull/3634/commits/2500c8cabdce95d546a21ba6e760099b4e9e6c28